api/envoy/api/v2/route/route.proto

```diff
@@ -437,6 +439,27 @@ message RouteAction {
     google.protobuf.Duration per_try_timeout = 3 [(gogoproto.stdduration) = true];
   }
 
+  // Specifies the idle timeout for the route. If not specified, this defaults
+  // to 5 minutes. The default value was select so as not to interfere with any
+  // smaller configured timeouts that may have existed in configurations prior
+  // to the introduction of this feature, while introducing robustness to TCP
+  // connections that terminate without FIN.
+  //
+  // The idle timeout is distinct to :ref:`timeout
```

Comment:
Perhaps also compare to connection idle timeout. Better, would it make sense to make a small timeout section in the HTTP arch overview or general config guide about timeouts, describe them all, and then link back to that section from all of these config options? It's only going to get more confusing as we add more configurable timeouts. Thoughts?

Reply:
Agreed. I'll take this as an action item for my followup when I introduce the global HCM idle timeout if that's OK.

Reply:
Sure SGTM.